reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-73.md

```diff
@@ -88,36 +81,36 @@ PowerShell 7.3-preview.8 includes the following features, updates, and breaking
 - Add `find.exe` to legacy argument binding behavior for Windows (#17715)
 - Add `-noprofileloadtime` switch to pwsh (#17535) (Thanks @rkeithhill!)
 
-For a complete list of changes, see the [Change Log][CHANGELOG] in the GitHub repository.
+For a complete list of changes, see the [Change Log][11] in the GitHub repository.
 
 ## Experimental Features
 
 PowerShell 7.3 introduces the following experimental features:
 
-- [PSExec][exp-psexec] - Adds the new `Switch-Process` cmdlet (alias `exec`) to provide `exec`
-  compatibility for non-Windows systems. In PowerShell 7.3-preview.8, this feature became
+- [PSExec][05] - Adds the new `Switch-Process` cmdlet (alias `exec`) to provide `exec`
+  compatibility for non-Windows systems. In PowerShell 7.3-rc.1, this feature became
```

Comment:
Clarifying: We're changing this version here because in the eventual `7.3.0` _non_ preview docs this should say `7.3.0`?

Reply:
My bad, this was a search/replace. But, yes, for GA we will change this to 7.3.0.